packages/bpk-component-theme-toggle/src/theming.js

```diff
@@ -20,12 +20,33 @@ import {
 } from 'bpk-tokens/tokens/base.es6';
 
 const bpkCustomTheme = {
+  buttonPrimaryTextColor: '#2d244c',
+  buttonPrimaryHoverTextColor: '#2d244c',
+  buttonPrimaryActiveTextColor: '#2d244c',
+  buttonPrimaryGradientStartColor: '#fce134',
+  buttonPrimaryGradientEndColor: '#f8c42d',
+  buttonPrimaryBackgroundColor: '#f8c42d',
+  buttonPrimaryHoverBackgroundColor: '#f8c42d',
+  buttonPrimaryActiveBackgroundColor: '#d99916',
+
+  buttonSecondaryTextColor: '#461963',
+  buttonSecondaryHoverTextColor: '#2D244C',
+  buttonSecondaryActiveTextColor: '#1A1331',
+  buttonSecondaryBorderColor: '#461963',
+  buttonSecondaryHoverBorderColor: '#2D244C',
+  buttonSecondaryActiveBorderColor: '#1A1331',
+  buttonSecondaryBackgroundColor: '#ffffff',
+  buttonSecondaryHoverBackgroundColor: '#ffffff',
+  buttonSecondaryActiveBackgroundColor: '#ffffff',
+
   blockquoteBarColor: '#461962',
+
   horizontalNavBarSelectedColor: '#461962',
   horizontalNavLinkActiveColor: '#2d244c',
   horizontalNavLinkColor: colorBlue500,
   horizontalNavLinkHoverColor: colorBlue500,
   horizontalNavLinkSelectedColor: '#491962',
+
   spinnerPrimaryColor: '#461962',
   barchartBarBackgroundColorNormal: '#865f9e',
```

Comment:
no new line here?

Reply:
yeah should be, i rebased and got more props it seems. :P